packages/create-plugin/templates/common/.config/Dockerfile

```
@@ -13,4 +13,4 @@ ENV GF_DEFAULT_APP_MODE "development"
 
 # Inject livereload script into grafana index.html
 USER root
-RUN sed -i 's/<\/body><\/html>/<script src=\"http:\/\/localhost:35729\/livereload.js\"><\/script><\/body><\/html>/g' /usr/share/grafana/public/views/index.html
\ No newline at end of file
+RUN sed -i 's/<\/body>/<script src=\"http:\/\/localhost:35729\/livereload.js\"><\/script><\/body>/g' /usr/share/grafana/public/views/index.html
```

I'm pretty sure what you have here will cover both 10.0.3 and 10.2.3. Did you test both cases locally? The only thing I might suggest is using `|` instead of `/` as the delimiter to make it easier to differentiate between them and the slashes in the URL.

```suggestion
RUN sed -i 's|</body>|<script src="http:\/\/localhost:35729\/livereload.js"><\/script>&|' /usr/share/grafana/public/views/index.html
```